tests/rbd/test_rbd_pwl_cache_cluster_operation.py

```diff
@@ -0,0 +1,190 @@
+"""RBD Persistent write back cache with cluster operations.
+
+Test Case Covered:
+CEPH-83574891 - Verify ceph cluster operations with persistent
+cache enabled in SSD mode with IO in-progress.
+
+Steps :
+1) Setup persistent write cache for an image
+2) Write data to the image using fio jobs check cache status
+3) Stop writing data to the image by killing the fio process, then start again
+4) Perform cluster operation like restarting mon, osd, overall ceph.target
+5) Remove mon node then add it back while IO keeps going
+6) Remove osd node then add it back while IO keeps going
+7) Initiate OSD scrub operation while IO keeps going
+8) check data corruption every time while IO keeps going
+
+Pre-requisites :
+- need client node with ceph-common package, conf and keyring files
+- FIO should be installed on the client.
+
+Environment and limitations:
+- The cluster should have 5 nodes + 1 SSD cache node
+- cluster/global-config-file: config/quincy/upi/octo-5-node-env.yaml
+- Should be Bare-metal.
+
+Support
+- Configure cluster with PWL Cache.
+- Only replicated pool supported, No EC pools."""
+
+import re
+
+from ceph.ceph_admin import CephAdmin
+from ceph.parallel import parallel
+from ceph.rados import utils
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.monitor_workflows import MonitorWorkflows
+
+# from ceph.utils import get_node_by_id
+from tests.rbd.rbd_peristent_writeback_cache import (
+    PersistentWriteAheadLog,
+    fio_ready,
+    get_entity_level,
+)
+from tests.rbd.rbd_utils import initial_rbd_config
+from tests.rbd.test_rbd_persistent_writeback_cache_flush import kill_fio
```

Comment:
If kill_fio method is being used in multiple test modules can we move if to tests.rbd.rbd_peristent_writeback_cache and import it both for this test case and for cache_flush test case?

Reply:
done